docs/eventing/samples/apache-camel-source/README.md

```diff
@@ -11,64 +10,75 @@ for generating events.
    any namespace where you want to run Camel sources.
 
    The preferred version that is compatible with Camel sources is
-   [Camel K v0.2.0](https://github.com/apache/camel-k/releases/tag/0.2.0).
+   [Camel K v0.3.3](https://github.com/apache/camel-k/releases/tag/0.3.3).
 
    Installation instruction are provided on the
    [Apache Camel K Github repository](https://github.com/apache/camel-k#installation).
    Documentation includes specific instructions for common Kubernetes
    environments, including development clusters.
 
-1. Install the Camel Source from the `camel.yaml` in the
-   [Eventing Sources release page](https://github.com/knative/eventing-contrib/releases):
+1. Download Kail binaries for Linux or OSX, which can be found on the [latest release](https://github.com/boz/kail/releases/latest) page. You can use `kail` instead of `kubectl logs` to tail the logs of the subscriber.
+
+1. Install the Camel Source from the `camel.yaml` in the [Eventing Sources release page](https://github.com/knative/eventing-contrib/releases):
 
    ```shell
    kubectl apply --filename camel.yaml
    ```
+   
 
-## Create a Channel and a Subscriber
+### Create a Channel and a Subscriber
 
-In order to check if a `CamelSource` is fully working, we will create:
+To verify that the `CamelSource` is working, we will create:
 
 - a simple Knative event display service that prints incoming events to its log
-- a in-memory channel named `camel-test` that will buffer events created by the
-  event source
-- a subscription to direct events from the test channel to the event display
-  service
+- an in-memory channel named `camel-test` that will buffer events created by the event source
+- a subscription to direct events from the test channel to the event display service
```

Comment:
You can unwrap these lines, but @mattmoor-sockpuppet  will come by and run prettier.io on them again.